thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -263,6 +265,7 @@ export default Ember.Controller.extend({
       Object.keys(updates).filter(urn => updates[urn]).forEach(urn => selectedUrns.add(urn));
       Object.keys(updates).filter(urn => !updates[urn]).forEach(urn => selectedUrns.delete(urn));
       this.set('selectedUrns', new Set(selectedUrns));
+      this.set('sessionModified', true);
```

Comment:
We can combine these 2 set statements and use `setProperties`

Reply:
fixed